code/src/components/Selector.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const Selector = ({ movieListType, setMovieListType }) => {
+  const onMovieListTypeChange = (event) => {
+    setMovieListType(event.target.value)
+  }
+  return (
+    <div className="drop-down-container">
+      <select className="drop-down" value={movieListType} onChange={onMovieListTypeChange}>
+        <option value="upcoming">Upcoming</option>
+        <option value="popular">Popular</option>
+        <option value="top_rated">Top Rated</option>
+      </select>
+    </div>
+  )
+}
```

Comment:
Great that you went width "select" to get different values, it both looks good and is very user-friendly.